src/v8/v8.cc

```diff
@@ -554,13 +585,19 @@ void V8::registerHostFunctionImpl(std::string_view module_name, std::string_view
       store_.get(), type.get(),
       [](void *data, const wasm::Val params[], wasm::Val[]) -> wasm::own<wasm::Trap> {
         auto func_data = reinterpret_cast<FuncData *>(data);
+        func_data->vm_->integration()->trace(
+            "[vm->host] " + func_data->name_ + "(" +
+            printValues(params, std::tuple_size<std::tuple<Args...>>::value) + ")");
```

Comment:
Is there any reason why this isn't using `printValues(params, sizeof...(Args)))` like the original code?

Reply:
no reason. just I didn't see the detail of the original impl. That looks much better, thanks!

Reply:
It's linked in the comment that you referenced here :P